doc/installpage.dox.in

```diff
@@ -144,7 +144,8 @@ pdlpack.py : INFO : > - sampling
 pdlpack.py : INFO : > - complete_linkage
 pdlpack.py : INFO : > - prediction_metrics
 pdlpack.py : INFO : > - balance_dataset
-pdlpack.py : INFO : PDL Tools @PDLTOOLS_VERSION_STRING@ installed successfully in PDLTOOLS schema.
+pdlpack.py : INFO : > - one_vs_rest
+pdlpack.py : INFO : PDL Tools 1.5 installed successfully in PDLTOOLS schema.
```

Comment:
Here as well, revert it back to "PDLTools @PDLTOOLS_VERSION_STRING@ was installed successfully in the PDLTools schema.